src/trigger-method.js

```diff
@@ -44,12 +44,16 @@ function triggerMethod(event, ...args) {
 //
 // e.g. `Marionette.triggerMethodOn(view, 'show')`
 // will trigger a "show" event or invoke onShow the view.
-function triggerMethodOn(context, ...args) {
-  var fnc = _.isFunction(context.triggerMethod) ?
-                context.triggerMethod :
-                triggerMethod;
+export function triggerMethodOn(context, ...args) {
+  var fnc = _.isFunction(context.triggerMethod) ? context.triggerMethod : triggerMethod;
+  return fnc.call(context, ...args);
```

Comment:
This should be `fnc.apply(context, args);` per https://github.com/marionettejs/backbone.marionette/issues/2849